src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx

```diff
@@ -295,7 +295,7 @@ class FormField extends React.Component {
                         onKeyDown: this.handleKeyDown,
                         onChange: this.handleChange,
                         onBlur: this.handleBlur,
-                        value,
+                        value: value === null ? '' : value,
```

Comment:
Fixes react warnings about input switching from controlled to uncontrolled

Reply:
this helped me understand what was going on, maybe worth including as a comment?